doc/Changelog.md

```diff
@@ -3,6 +3,11 @@ Changelog {#Changelog}
 
 # git master
 
+* [115](https://github.com/BlueBrain/Brion/pull/113):
+  Changes and fixes in brian Python module:
```

Comment:
That reminds me that I have to build everything to see which code the change breaks.